packaging/linux/openra.appimage.in

```diff
@@ -43,21 +46,23 @@ fi
 
 # Run the game
 export SDL_VIDEO_X11_WMCLASS="openra-{MODID}-{TAG}"
-./OpenRA Game.Mod={MODID} Engine.LaunchPath="${LAUNCHER}" "${JOIN_SERVER}" "$@"
+./OpenRA Game.Mod={MODID} Engine.LaunchPath="${LAUNCHER}" "${JOIN_SERVER}" "$@" && rc=0 || rc=$?
 
 # Show a crash dialog if something went wrong
-if [ $? != 0 ] && [ $? != 1 ]; then
+if [ $rc != 0 ] && [ $rc != 1 ]; then
 	LOGS="${XDG_CONFIG_HOME:-${HOME}/.config}/openra/Logs"
 	if [ ! -d "${LOGS}" ] && [ -d "${HOME}/.openra/Logs" ]; then
 		LOGS="${HOME}/.openra/Logs"
 	fi
 
-	test -d Support/Logs && LOGS="${PWD}/Support/Logs"
+	if [ -d Support/Logs ]; then
```

Comment:
trivial: executable .. `-x` over `-d`

also: `"./Support/Logs"`.

is this a way to override the log location? for developers, or on macos?

Reply:
> executable .. `-x` over `-d`

But it has to be a directory, right? What would be the point of this?

Reply:
> > executable .. `-x` over `-d`
> 
> But it has to be a directory, right? What would be the point of this?

trivial. even it its a directory it won't do you much good if you do not have executable rights on it.

Reply:
> even it its a directory it won't do you much good if you do not have executable rights on it.

It doesn't seem meaningful. Using `-d` is good enough. Just checking with `-x` still doesn't mean that you can actually create (or write existing) log files and it has the additional caveat that it succeeds for executable files.